src/main/java/step3/domain/BonusBall.java

```diff
@@ -0,0 +1,22 @@
+package step3.domain;
+
+import java.util.List;
+
+public class BonusBall {
+    private int number;
```

Comment:
```suggestion
    private Lotto number;
```

보너스 넘버 또한 1~45의 값을 검증해보면 좋겠습니다 :)

Reply:
수정하였습니다!
Lotto를 만들어 넣고, 여기서도 활용을 할 수 있다는 생각을 못하였네요
감사합니다!